main/lsp/LSPPreprocessor.cc

```diff
@@ -92,16 +78,9 @@ void LSPPreprocessor::mergeFileChanges(absl::Mutex &mtx, QueueState &state) {
     ENFORCE(pendingRequests.size() + requestsMergedCounter == originalSize);
 }
 
-unique_ptr<LSPMessage> LSPPreprocessor::makeAndCommitWorkspaceEdit(unique_ptr<SorbetWorkspaceEditParams> params,
```

Comment:
Previously, we converted all textDocument/didOpen / didChange / didClose / watchman updates into a new LSPMessage with method sorbet/workspaceEdit.

We no longer need to do this; we translate all of them directly into a SorbetWorkspaceEditTask instead.